tests/template_tests/test_extends.py

```diff
@@ -117,3 +119,11 @@ def test_unique_history_per_loader(self):
         template = engine.get_template('base.html')
         output = template.render(Context({}))
         self.assertEqual(output.strip(), 'loader2 loader1')
+
+    @override_settings(INSTALLED_APPS=[
+        'template_tests.extended',
+        'template_tests.base',
+    ])
+    def test_get_extended_base_template_w_extended_included_template(self):
```

Comment:
w-> with?
Adding a docstring to the test would be useful as its purpose isn't trivial to understand.